core/windows_and_tabs/window_snap.py

```diff
@@ -316,17 +319,39 @@ def window_split_position(m) -> Dict[int, list[RelativeScreenPos]]:
 
 @mod.action_class
 class Actions:
-    def snap_window(position: RelativeScreenPos) -> None:
-        """Move the active window to a specific position on its current screen, given a `RelativeScreenPos` object."""
-        _snap_window_helper(ui.active_window(), position)
+    def snap_window(
+        position: RelativeScreenPos, window: Optional[Window] = None
+    ) -> None:
+        """Move a window (defaults to the active window) to a specific position on its current screen, given a `RelativeScreenPos` object."""
+        if window is None:
+            window = ui.active_window()
+        _snap_window_helper(window, position)
 
-    def snap_window_to_position(position_name: str) -> None:
-        """Move the active window to a specifically named position on its current screen, using a key from `_snap_positions`."""
-        actions.user.snap_window(_snap_positions[position_name])
+    def snap_window_to_position(
+        position_name: str, window: Optional[Window] = None
+    ) -> None:
+        """Move a window (defaults to the active window) to a specifically named position on its current screen, using a key from `_snap_positions`."""
+        position: Optional[RelativeScreenPos] = None
+        if position_name in _snap_positions:
+            position = _snap_positions[position_name]
+        else:
+            # Previously this function took a spoken form, but we now have constant identifiers in `_snap_positions`.
+            # If the user passed a previous spoken form instead, see if we can convert it to the new identifier.
+            new_key = actions.user.formatted_text(position_name, "ALL_CAPS,SNAKE_CASE")
+            if new_key in _snap_positions:
+                actions.user.deprecate_action(
+                    "2024-12-02",
+                    f"snap_window_to_position('{position_name}')",
+                    f"snap_window_to_position('{new_key}')",
+                )
+                position = _snap_positions[new_key]
+                actions.user.snap_window(position, window)
+            else:
+                raise KeyError()
 
     def move_window_next_screen() -> None:
         """Move the active window to a specific screen."""
-        _move_to_screen(ui.active_window(), offset=1)
+        actions.user.snap_window_to_position("top right", ui.active_window())
```

Comment:
Not sure why you are using "top right" here. This is only supposed to move to another screen not move the window relative to the screen? Even if this is intentional, we are wondering why you are using the words "top right" instead of an identifier as you had implemented in this PR?

Reply:
Whoops - that was test code I didn't mean to commit. Will remove-was testing the fall back